### ISSUE TITLE
Have you reproduced results mentioned in the paper

### ISSUE THREAD

**Issue by shaojunyao** [chenjiasheng/mwer#1]
thanks for sharing implementation of mwer, Have you reproduced results mentioned in the paper of Rohit Prabhavalkar etc.
    MINIMUM WORD ERROR RATE TRAINING FOR ATTENTION-BASED
    SEQUENCE-TO-SEQUENCE MODELS

**Comment by chenjiasheng** [comment]
My baseline model is very different from the one in this paper.
However, over-fit experiment shows that the loss function works well.
